internal/modules/auth/test.py

```diff
@@ -15,25 +34,246 @@ class TestAuthModuleTxsQueries(unittest.TestCase):
     which test queries of auth module
     """
 
+    @classmethod
+    def setUpClass(cls):
+        """
+        The function `setUpClass` will be called before running any test case.
+        """
+        status, multisigaccount = keys_add("multisigaccount", multisig=True)
+        assert status, "Failed to add multisig account"
+        multi_key_address = multisigaccount["address"]
+
+        status, account1 = keys_show("account1")
+        assert status, "Failed to get account1 account details"
+        account1_address = account1["address"]
```

Comment:
use shorter namings. 
```suggestion
        acc1_addr = acc1["address"]
```